package.json

```diff
@@ -9,7 +9,8 @@
     "prepublish": "npm run build"
   },
   "bin": {
-    "jspmjasmine": "./bin/jspmjasmine.js"
+    "jspmjasmine": "./bin/jspmjasmine.js",
+    "jspmjasmineinspect": "./bin/jspmjasmineinspect.js"
```

Comment:
The only way I can think of to get this feature is to have a whole new executable.